cdap-common/src/main/java/co/cask/cdap/common/metrics/MetricsConstants.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright © 2014 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.common.metrics;
+
+/**
+ * Constants for metrics.
+ */
+public interface MetricsConstants {
```

Comment:
Just to simplify using it. I think there's probably a better place to put this though, but I don't know it.